core/src/main/scala/org/apache/spark/deploy/FaultToleranceTest.scala

```diff
@@ -187,7 +188,7 @@ private object FaultToleranceTest extends App with Logging {
       fn
       numPassed += 1
       logInfo("==============================================")
-      logInfo("Passed: " + name)
+      logInfo(log"Passed: ${MDC(TEST_NAME, name)}")
```

Comment:
We don't need to migrate the log entries from tests. Let's revert the changes in this file to avoid introducing new log keys.